### PR TITLE
Fix/ps 522

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
1) Content identifier added in the publish screen. 
2) Pdf thumbnail issue resolved.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -53,12 +53,12 @@ android {
     }
 
     signingConfigs {
-        debug  {
-            storeFile file('keystore.jks')
-            storePassword 'tech.pylons'
-            keyAlias 'key_tech_pylon'
-            keyPassword 'tech.pylons'
-        }
+//        debug  {
```
Comment (mustafapylons) [inline android/app/build.gradle:56]: Is this dead code necessary for testing?

File: lib/screens/creator_hub/widgets/draft_list_tile.dart [hunk 2]
```diff
@@ -63,49 +64,47 @@ class DraftListTile extends StatelessWidget {
...
-                  )
-                ),
+                      onVideo: (context) => CachedNetworkImage(
+                        fit: BoxFit.fill,
+                        imageUrl: nft.thumbnailUrl.changeDomain(),
+                        errorWidget: (a, b, c) => const Center(child: Icon(Icons.error_outline)),
+                        placeholder: (context, url) => Shimmer(color: EaselAppTheme.cardBackground, child: const SizedBox.expand()),
+                      ),
+                      onPdf: (context) => CachedNetworkImage(
+                        fit: BoxFit.fill,
```
Comment (mustafapylons) [inline lib/screens/creator_hub/widgets/draft_list_tile.dart]: All three cases are resolving into the same code block, reusable code would be better.

**Review by mustafapylons — COMMENTED** [review]
Few fixes, otherwise code is good.

**Review by Amean — APPROVED** [review]

File: android/app/build.gradle (hunk 1 above)
Comment (kjawadDeveloper2) [reply android/app/build.gradle:56]: It is needed when we release the build.

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Pull request merged**